libs/blocks/notification/notification.js

```diff
@@ -65,7 +67,8 @@ function getBlockData(el) {
   const variant = variants.find((varClass) => el.classList.contains(varClass)) || defaultVariant;
   const size = sizes.find((sizeClass) => el.classList.contains(sizeClass)) || defaultSize;
   const fontSizes = [...blockConfig[variant][size]];
-  if (el.classList.contains('s-button')) fontSizes.splice(3, 1, 'm');
+  const buttonSize = el.className.match(/([xsml])+-button/);
+  if (buttonSize) fontSizes.splice(3, 1, buttonSize[1]);
```

Comment:
Wouldn't this use `s` for `xs-button` and `l` for `xl-button`? Is that the intention?

Reply:
D'oh, right you are. I put my `+` in the wrong place 😅